package.json

```diff
@@ -49,7 +49,7 @@
         "@elementor/externalize-wordpress-assets-webpack-plugin": "0.1.0",
         "@elementor/extract-i18n-wordpress-experssions-webpack-plugin": "0.1.0",
         "@elementor/generate-wordpress-asset-file-webpack-plugin": "0.1.0",
-        "@playwright/test": "^1.33.0",
+        "@playwright/test": "1.33.0",
```

Comment:
why?

Reply:
We don't want surprises when we run `npm install`, this is a more stable approach

Reply:
also `^` breaks docker to run` test:playwright:elements-regression:linux`